server/app/routes/v1/audit_logs_api.rb

```diff
@@ -0,0 +1,21 @@
+module V1
+  class AuditLogsApi < Roda
+    include TokenAuthenticationHelper
+    include CurrentUser
+    include RequestHelpers
+    include Auditor
+
+    plugin :streaming
+    
+    route do |r|
+      r.get do
+        halt(403, 'Access denied') unless current_user.master_admin?
+
+        limit = request.params['limit'] || 500
```

Comment:
Yes, but at least we need to plug this "bug" so that server won't crash if too many logs are requested? Refactor to @SpComb implementation should be in a separate PR and not here.